lib/linguist/languages.yml

```diff
@@ -3781,6 +3781,16 @@ Monkey:
   ace_mode: text
   tm_scope: source.monkey
   language_id: 236
+Monkey C:
+  type: programming
+  color: "#8D6747"
+  extensions:
+  - ".mc"
```

Comment:
There appear to be a lot of other users of the `.mc` extension that don't all appear to be Monkey C and Linguist doesn't know about this extension yet, so they're all going to be classified as Monkey C. I think we're going to need to identify the language of at least one other user of this extension and add it, possibly with a heuristic too.